docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,175 @@
+               dlog_print(DLOG_INFO, LOG_TAG, "rejected");
+               break;
+           default:
+               break;
+   }
+
+   void
+   connect_autofill_daemon(autofill_manager_h amh)
+   {
+       int ret;
+       ret = autofill_manager_connect(amh, manager_connection_status_changed_cb, NULL);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+   You can use the autofill manager functions after receiving `AUTOFILL_MANAGER_CONNECTION_STATUS_CONNECTED` in `manager_connection_status_changed_cb()`.
+
+## Get Installed Autofill Service List
+
+To get the installed autofill service list:
+
+1. To access Autofill service information, create a callback function using the `autofill_service_info_cb()` function, which is invoked for each installed Autofill service application:
+
+   ```c
+   #include <app_manager.h>
+
+   static bool autofill_service_info_cb(const char* app_id, void* user_data)
+   {
+       app_info_h app_h;
+       char *label = NULL;
+
+       app_info_create(app_id, &app_h);
+       app_info_get_label(app_h, &label);
+
+       dlog_print(DLOG_INFO, LOG_TAG, "app id : %s, label : %s", app_id, label);
+
+       if (label)
+           free(label);
+
+       app_info_destroy(app_h);
+
+       return true;
+   }
+   ```
+
+2. To get the installed Autofill service by invoking a callback function for each installed Autofill service application, use the `autofill_manager_foreach_autofill_service()` function:
+
+   ```c
+    ret = autofill_manager_foreach_autofill_service(amh, autofill_service_info_cb, NULL);
+    if (AUTOFILL_ERROR_NONE != ret)
+        /* Error handling */
+   ```
+
+## Retrieve Current Autofill Service
+
+To get the application ID of the current used autofill service, use the `autofill_manager_get_autofill_service()` function:
+
+```c
+void get_autofill_service(autofill_manager_h amh)
+{
+    char *active_autofill_service_app_id = NULL;
+    int ret = autofill_manager_get_autofill_service(g_amh, &active_autofill_service_app_id);
+    if (AUTOFILL_ERROR_NONE == ret) {
+        dlog_print(DLOG_INFO, LOG_TAG, "app id : %s", active_autofill_service_app_id);
+        if (active_autofill_service_app_id) {
+            free(active_autofill_service_app_id);
+        }
+    }
+}
+```
+
+## Change Autofill Service
+
+To change autofill service as the other autofill service, use the `autofill_manager_set_autofill_service()` function:
+
+```c
+void set_autofill_service(autofill_manager_h amh, const char *autofill_service_app_id)
+{
+    dlog_print(DLOG_INFO, LOG_TAG, "index : %d", index);
+
+    int ret = autofill_manager_set_autofill_service(amh, autofill_service_app_id);
+    if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+}
+```
+
+## Release Resources
+
+Incase, the Autofill manager library is not needed in your application, you must destroy the Autofill manager handle using the `autofill_manager_destroy()`  function:
```

Comment:
`autofill_manager_destroy()`  function
Remove white space between "`autofill_manager_destroy()`" and "function"

Reply:
How about:
"After you have finished working with the Autofill manager library, destroy the the Autofill manager handle:"

Reply:
@mijin85cho, it can be changed as requested:
After you have finished working with the Autofill manager library, destroy the Autofill manager handle: